tss/.htaccess

```diff
@@ -0,0 +1,28 @@
+# RDF Time Series Snippets (TSS)
+RewriteEngine on
+DirectorySlash Off
+
+## TSS Ontology content negotiation resources
+RewriteCond %{HTTP_ACCEPT} text/html
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.html#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} application/ld\+json
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.jsonld#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} application/n-triples
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.nt#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.rdf#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} text/turtle
+RewriteRule ^(.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/tss.ttl#$1 [NE,R,L]
+RewriteCond %{HTTP_ACCEPT} .+
+
+RewriteRule (.*)\.(tss).conneg$ https://knowledgeonwebscale.github.io/rdf-timeseriessnippets/406.html [NE,L,R=406]
+
+# === TSS ===
+RewriteCond %{REQUEST_URI} tss
+RewriteRule tss/?$ https://knowledgeonwebscale.github.io/rdf-timerseriessnippets/tss.ttl [NE,R,L]
```

Comment:
Patterns are relative to the dir.  Did you mean to match https://w3id.org/tss/tss?
Doesn't look like the RewriteConds with same match are needed?
Same below.

Reply:
I adjusted it as I want to match to `https://w3id.org/tss/` here.